### DISCUSSION TITLE
Prisma support native database

### DISCUSSION THREAD

**Discussion by skyward06** [prisma/prisma#25378] [Q&A] [answered]
### Question

Does Prisma support native database sharding or partitioning for scaling large databases?

**Comment by hadez0528** [comment, marked as answer]
Currently, Prisma doesn't offer built-in support for database sharding or partitioning. However, you can manage sharding at the database level and use multiple Prisma clients to interact with different shards. Be mindful of handling distributed queries and ensuring that your application can effectively route requests to the correct shard.
